src/app/component/logout/logout.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-logout',
+  templateUrl: './logout.component.html',
+  styleUrls: ['./logout.component.scss']
+})
+export class LogoutComponent {
+
+
+  
+  constructor(private router:Router){}
+
+  logout(): void{
```

Comment:
when we exit from the page , and click logout - removeItem is missing

Reply:
fixed